components/link/link.js

```diff
@@ -118,12 +130,19 @@ class Link extends LocalizeCoreElement(FocusMixin(LitElement)) {
 						display: none;
 					}
 				}
+
+				a[aria-disabled="true"] {
```

Comment:
Wondering if it makes sense to move this into `linkStyles`? That way someone using those but not `<d2l-link>` could leverage them. Of course they'd need to implement all their own click/`href` logic, so maybe not worth it. But it would let us backport this to MVC. 🤔